package.json

```diff
@@ -194,6 +191,7 @@
     "form-data": "^4.0.0",
     "got": "^11.8.1",
     "graphviz": "^0.0.9",
+    "husky": "^4.3.8",
```

Comment:
Was the only way to get commit lint passing (without it it prompted me the following error: `npm ERR! could not determine executable to run`)

cc @ehmicky